parser-typechecker/src/Unison/Codebase/TranscriptParser.hs

```diff
@@ -370,8 +370,8 @@ expectingError = isJust <$> optional (word ":error")
 untilSpace1 :: P Text
 untilSpace1 = P.takeWhile1P Nothing (not . Char.isSpace)
 
-lineUntilSpace :: P Text
-lineUntilSpace = P.takeWhileP Nothing (\ch -> ch `elem` (" \t" :: String))
+language :: P Text
+language = P.takeWhileP Nothing (\ch -> Char.isDigit ch || Char.isLower ch || ch == '_' )
```

Comment:
This might be overly conservative; e.g. should it have to be all lowercase?

Reply:
Is there some list or spec somewhere online we can use as reference?  I've been looking but haven't found one.

Reply:
Well maybe this is fine. 😅

Reply:
I mentioned that through some digging, it looks like for language highlighting to work (as supported by github at least), the `fenceType` of the block must match the names of the files in this repo. https://github.com/ajaxorg/ace/tree/master/lib/ace/mode 

From what I can tell, there are only lowercase names, with very few that contain a digit or an underscore. Definitely no symbols or punctuation. In reality, most people writing a blog via the transcript parser will probably care to compare with a "haskell/agda/scala" etc block.